src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1358,7 +1358,6 @@ const static EnumMap<GameCommand> ActionNameToType = {
     { "bannersetname", GameCommand::SetBannerName },
     { "bannersetstyle", GameCommand::SetBannerStyle },
     { "clearscenery", GameCommand::ClearScenery },
-    { "climateset", GameCommand::SetClimate },
```

Comment:
Was the plugin version bumped when climate objects were introduced? (I.o.w. do plugins have a way to know when not to call this?)

Reply:
Not specifically, no. In the mean time, the plugin version has been bumped by #23359, though, so fortunately that provides a reference point.